src/App.vue

```diff
@@ -111,6 +112,8 @@
           :screen="screen"
           title="Default"
           :render-controls="displayBuilder"
+          :show-templates-panel="showTemplatesPanel"
+          :reshow-templates-panel="reshowTemplatesPanel"
```

Comment:
Hey @mcraeteisha do we still need to use the `reshowTemplatePanel`? I do not see it being used in the `vue-form-builder` component.